e2e/worker/frontend/server.js

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+'use strict';
+
+const express = require('express');
+const { SNSClient, PublishCommand } = require("@aws-sdk/client-sns");
+
+const PORT = 8080;
+const HOST = '0.0.0.0';
+const client = new SNSClient({ region: process.env.AWS_DEFAULT_REGION });
+const app = express();
+
+// Start the service waiting for an ack message.
+let status = 'waiting on acknowledgement';
+
+// Each health check request from the ALB will result in publishing an event.
+app.get('/', async (req, res) => {
+  const {events} = JSON.parse(process.env.COPILOT_SNS_TOPIC_ARNS);
+  const out = await client.send(new PublishCommand({
+    Message: "healthcheck",
+    TopicArn: events,
+  }));
```

Comment:
Is it a problem if we accumulate too many messages in a queue before the worker service is able to consume them?

Reply:
Nope! SQS is cool 😎 

> A single Amazon SQS message queue can contain an unlimited number of messages. However, there is a 120,000 quota for the number of inflight messages for a standard queue and 20,000 for a FIFO queue. Messages are inflight after they have been received from the queue by a consuming component, but have not yet been deleted from the queue.